openvdb/CMakeLists.txt

```diff
@@ -145,6 +145,14 @@ set(OPENVDB_CORE_DEPENDENT_LIBS
   IlmBase::Half
 )
 
+if(WIN32)
+  if(OPENVDB_DISABLE_BOOST_IMPLICIT_LINKING)
+      list(APPEND OPENVDB_CORE_DEPENDENT_LIBS
+        Boost::disable_autolinking  # add -DBOOST_ALL_NO_LIB
```

Comment:
I wouldn't mind a quick comment explaining what this is for and why we need it?

Reply:
Boost headers contain #pragma commands on Windows which causes Boost libraries to be linked in automatically when including the appropriate header. Depending on how you're building boost, these naming conventions don't always match causing linker errors. This target turns that feature off. I'll add this comment.